src/components/open-api-page/docs.mdx

```diff
@@ -0,0 +1,65 @@
+---
+componentName: 'OpenApiPage'
```

Comment:
Note: the `components/open-api-page` directory is a localized version of [the same component from react-components](https://github.com/hashicorp/react-components/tree/main/packages/open-api-page). I've made minimal changes in order to get this stub working; intent is to properly finish implementing this localized component once we have a final design spec.